### ISSUE TITLE
Copy change for prompt when downloading insecurely

### ISSUE THREAD

**Issue by Uni-verse** [brave/brave-browser#36032]
## Description 
Copy change for prompt when downloading insecurely

Message: `File can't be downloaded securely`
Message body: `(X KB)`
Action: `Discard` / `Keep`

**Note:** This is a new feature in C122

## Steps to reproduce 

   1. Enter `http://www.pdf995.com/samples/pdf.pdf` to URL bar or attempt to download file from insecure URL 
   2. Observe prompt
   3. 

**QA Test Plan:** 

1. Verify copy change (TBD)
2. Confirm file is discarded when user selects `Discard`
3. Confirm file is kept when user selects `Keep`
4. Check fresh / upgraded profile.

## Actual result 
![Screenshot 2024-02-13 at 4 32 07 PM](https://github.com/brave/brave-browser/assets/17885425/37baa941-9126-4f54-b919-f8e9611bbae3)

## Expected result
Change copy for prompt above

## Issue reproduces how often 
Always

## Version/Channel Information:

- Can you reproduce this issue with the current Play Store version? No
- Can you reproduce this issue with the current Play Store Beta version? No
- Can you reproduce this issue with the current Play Store Nightly version? Yes

## Device details
   - Install type (ARM, x86): ARM
   - Device type (Phone, Tablet, Phablet): Samsung GS 21
   - Android version: 13
   
## Brave version
`1.64.69, Chromium 122.0.6261.29`

**Comment by fmarier** [comment]
I'd suggest something like:
> File is not provided securely by the site via HTTPS. It can be downloaded using insecure HTTP.

**Comment by rmcfadden3** [comment]
@fmarier — based on your version, it sounds like the thing to note here is that this is NOT a Brave failing. It's rather that the site itself isn't making the file available via HTTPS. If I'm reading that correctly, then I might slightly tweak your copy to:

`Site is not providing the file for secure HTTPS download. Proceed with insecure HTTP download instead?`

wdyt?

**Comment by fmarier** [comment]
@rmcfadden3 That's correct. The site is to blame for this.

Your version sounds good to me :+1:

**Comment by anthonypkeane** [comment]
Thanks @rmcfadden3 

Can I make one small suggestion


{Site Name or URL] is not providing the file for secure HTTPS download. Proceed with insecure HTTP download instead?
Cancel | Download 

The URL only needs to be the domain name, i.e github.com

@fmarier does this sound okay? Thanks

**Comment by AntiSol** [comment]
> [@rmcfadden3](https://github.com/rmcfadden3) That's correct. The site is to blame for this.

No, the site is not "to blame" for this - brave changed to disallow downloading files via HTTP. HTTP has been a standardised protocol for 30+ years. Serving files over HTTP is not "a bug". This is security theatre and only serves to add friction to users' experience without creating any safety as far as I can see.
